pkg/cache/application.go

```diff
@@ -628,6 +647,22 @@ func (app *Application) handleReleaseAppAllocationEvent(event *fsm.Event) {
 			}
 		}
 	}
+	app.postAppReleased(eventArgs)
```

Comment:
hi @manirajv06 sorry, I did not mean to add `eventArgs` to this function, this is not needed. I forgot to delete that from my copy-paste code, sorry about that. Can you remove this arg and keep the function as simple as it was? thanks!